### ISSUE TITLE
suport for centimeters

### ISSUE THREAD

**Issue by musketyr** [craigburke/document-builder#19]
Apart from inches and pixels the `UnitCategory` should support centimeters as well,

**Comment by craigburke** [comment]
Sure. That should be a pretty easy thing to add.

**Comment by goeh** [comment]
I just want to let you know that I think I fixed centimeter/cm in my fork https://github.com/goeh/document-builder/tree/iso-216-metric
This week I'm assigned a task to implement/produce a pretty complex Word document in a Grails application, and I'm counting on document-builder to deliver ;-). I will start sending pull requests in a day or two, so stay tuned...

**Comment by craigburke** [comment]
@goeh cool! I actually ended up adding this in 0.4.15 a bit ago but forgot to push those changes. Even though it's not in the documentation yet I guess I can close this one.

**Comment by craigburke** [comment]
@goeh the project definitely still has some gaps so hopefully it works for you. Looking forward to the PRs. 

Feel free to ping me beforehand or file an issue if you'd like feedback before you start any work on your end.

**Comment by goeh** [comment]
Thanks. I have pretty long experience with different libraries for producing documents (jasper, itext, windward, pdfbox, etc.) in my previous life, so I know how hard it is to get these thing right.
The library that I used most recently was https://github.com/ullenboom/jrtf but the suite of (Word+PDF) reports I want to produce now need more fine grained control of line spacing, indents, etc. and I could not tweak jrtf to my needs.
I've been following your work with document-builder from a distance and last week I took a dive into your source. It made me even more convinced that it's a great library to build upon. I hope I'm right. :)
